test/WebJobs.Script.Tests/TestScripts/CSharp/QueueTriggerToBlob/run.csx

```diff
@@ -8,7 +8,7 @@ public static void Run(WorkItem input, out string output, TraceWriter log)
 {
     string json = string.Format("{{ \"id\": \"{0}\" }}", input.Id);
 
-    log.Verbose($"C# script processed queue message. Item={json}");
+    log.Info($"C# script processed queue message. Item={json}");
```

Comment:
Note that for C# this is actually a behavioral change - the default level is info, so any Verbose logs will no longer show up in the log stream by default.

I'm going to send a PR to update all the C# templates.